backend/vaa-strapi/src/extensions/users-permissions/strapi-server.js

```diff
@@ -6,7 +6,13 @@ const defaultPermissions = [
   {action: 'plugin::users-permissions.candidate.check', roleType: 'public'},
   {action: 'plugin::users-permissions.candidate.register', roleType: 'public'},
   {action: 'api::candidate.candidate.findOne', roleType: 'authenticated'},
-  {action: 'api::candidate.candidate.find', roleType: 'authenticated'}
+  {action: 'api::candidate.candidate.find', roleType: 'authenticated'},
```

Comment:
Minor nitpick but for consistency can we have the order match the other permissions added, so all permissions are in the order of:
- find
- findOne